app/request_handlers.py

```diff
@@ -989,7 +981,8 @@ async def get(self, request):
 
         self.log_entry(request, display_region + '/request/' + request_type + '/code-sent-by-post')
 
-        attributes = await self.get_check_attributes(request, request_type)
+        session = await get_existing_session(request, 'requests', request_type)
+        attributes = get_session_value(session, 'attributes', 'requests', request_type)
```

Comment:
Where these 2 calls have 'requests' should be 'request'. This applies throughout request_handlers

Reply:
Updated throughout. Shows how long ago I started this change.